src/developer-tools/get-started/set-up.html

```diff
@@ -45,26 +44,38 @@ <h4>CREATE AN ACCOUNT AT HTTPS://RAFIKI.MONEY</h4>
           </ol>
           <h3>2. Locate Your Own Payment Pointer</h3>
           <ol>
-            <li>In each wallet, you will see a page with a payment pointer in the following formats:&nbsp;<code>$xpring.money/{your-account-name}</code>&nbsp;(RippleX Wallet) and&nbsp;<code>rafiki.money/p/{user-id}</code>&nbsp;(Rafki Money). As indicated by the payment pointer, each wallet is Interledger-enabled.</li>
-            <li>Because your new accounts don't have any test funds, you will want to add value to them. In RippleX, click the blue&nbsp;<code>Make it rain</code>&nbsp;button to get free money! In Rafki, click the&nbsp;<code>Add Funds</code>&nbsp;button.</li>
+            <li>In each wallet, you will see a page with a payment pointer in the following formats:&nbsp;<code>$jc.ilpv4.dev/{your-account-name}</code>&nbsp;(RippleX Wallet) and&nbsp;<code>rafiki.money/p/{user-id}</code>&nbsp;(Rafki Money). As indicated by the payment pointer, each wallet is Interledger-enabled.</li>
+            <li>Because your new accounts don't have any test funds, you will want to add value to them. {{DAVID_TODO FOR RIPPLEX}}. In Rafki, click the&nbsp;<code>Add Funds</code>&nbsp;button.</li>
```

Comment:
I'm not sure what `{{DAVID_TODO FOR RIPPLEX}}` is.

Reply:
Thanks @AlexLakatos , actually I forgot to write that this PR is still WIP.
I am not sure what the ILP faucet is to fund an account created on https://faucet.ilpv4.dev/.
Do you know what the cURL request is? Or the faucet account ID?
That's why I wrote `{{DAVID_TODO FOR RIPPLEX}}`, David is in my team, he is my manager and might know the faucet to use to fund an ILP account.

Reply:
I'm not sure about that either I'm guessing that's @sappenin and he has the answer here 😄

Reply:
I fixed these links.